src/reaper_osara.cpp

```diff
@@ -3,6 +3,7 @@
  * Main plug-in code
  * Author: James Teh <jamie@jantrid.net>
  * Copyright 2014-2019 NV Access Limited, James Teh
+ * Copyright 2019-2021 James Teh
```

Comment:
Strictly speaking, having multiple copyright entries is probably more accurate, but I think that's going to get unmanageable with time, so I'm condensing.

```suggestion
 * Copyright 2014-2021 NV Access Limited, James Teh
```